packages/components/src/Disclosure/Disclosure.css

```diff
@@ -9,13 +9,17 @@
 }
 
 .summary {
-  padding: var(--space-smaller) 0 var(--space-smallest) 0;
+  --disclosure--interactionOffset: var(--space-smaller);
```

Comment:
This might be a shade too clever but wanted to explicitly tie this value to the left spacing and the negative offset

Reply:
@chris-at-jobber could we go even further here and use `small` instead? The space is good but it's still pretty tight.

Reply:
Just poking around, I think we could - might add a bit of the same to the right side in that case

![image](https://github.com/user-attachments/assets/b5e95112-7527-4bc8-bfc1-dc0e1c23cb3d)
![image](https://github.com/user-attachments/assets/7dbf1503-64c9-465c-95c7-c9891f40078d)

![image](https://github.com/user-attachments/assets/c6410e01-feb5-4d09-beb1-68bcbd483b77)
![image](https://github.com/user-attachments/assets/779d86e7-17ae-4f9e-aae7-080dece3bc79)

It does bug me a _bit_ when the hovered surface/focus ring is more noticeably misaligned to the content, but those states are fleeting and the component itself looks cleaner - thoughts?

Reply:
I don't love it for a few reasons
- less obvious what area is actually interactive
- inconsistent with the majority of our "hoverable" interactive elements (Button, List, DataList, Menu items, Combobox, Chip, etc)
- since teams can pass in a `{child}` as the title, would create either inconsistent or unpredictable interactive states (both for consuming teams and end users)

Reply:
@chris-at-jobber k, lets just do more padding on each side?